lambda-deployment-orca/src/main/java/com/amazon/aws/spinnaker/plugin/lambda/traffic/LambdaTrafficUpdateVerificationTask.java

```diff
@@ -63,7 +73,47 @@ public TaskResult execute(@Nonnull StageExecution stage) {
             return formErrorTaskResult(stage,op.getErrors().getMessage());
         }
 
+        if (!DeploymentStrategyEnum.$WEIGHTED.toString().equals(stage.getContext().get("deploymentStrategy"))) {
+            boolean inValid = validateWeights(stage);
```

Comment:
Doesn't this return true when things are good?  So then:
```suggestion
            boolean valid = validateWeights(stage);
```
?

Reply:
Nope its the other way around, let me change the javadoc